app/routes/public/cfs.js

```diff
@@ -1,15 +1,21 @@
 import Ember from 'ember';
+import moment from 'moment';
 
-const { Route, RSVP } = Ember;
+const { Route, RSVP, computed } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Call for Speakers');
   },
 
   model() {
+    const eventDetails = this.modelFor('public');
     return RSVP.hash({
-      event: this.modelFor('public')
+      event        : eventDetails,
+      speakersCall : eventDetails.get('speakersCall'),
+      isOpen       : computed('eventDetails.endsAt', 'speakersCall.endsAt', function() {
+        return moment(this.get('eventDetails.endsAt')).isAfter(this.get('speakersCall.endsAt'), 'year');
```

Comment:
But then a speakers call almost always closes _before_ an event even starts. Thats that why a speakers call has its own `startsAt` and `endsAt`. You should only be using those two for checking.